src/main/java/frc/robot/subsystems/Rollergripper.java

```diff
@@ -14,24 +14,24 @@
 import static frc.robot.Constants.RollerGripperConstants.*;
 
 public class Rollergripper extends SubsystemBase {
-  private final CANSparkMax rightRoller = new CANSparkMax(RIGHT_ROLLER_MOTOR_ID, kBrushless);
-  private final CANSparkMax leftRoller = new CANSparkMax(LEFT_ROLLER_MOTOR_ID, kBrushless);
+  private final CANSparkMax FollowRoller = new CANSparkMax(RIGHT_ROLLER_MOTOR_ID, kBrushless);
+  private final CANSparkMax LeadRoller = new CANSparkMax(LEFT_ROLLER_MOTOR_ID, kBrushless);
```

Comment:
variables should start with a lower case letter

Reply:
also when having one motor follo another there's no need to call the leader "leader"..
just call it something like rollers or motor \ motors, and then you can do.
motors.set() and it just looks better and makes more sense even to someone who isn't familiar with our code

Reply:
yeah but then its just "roller"
seems kinda "empty" in a certain way